bower.json

```diff
@@ -11,7 +11,7 @@
     "libs"
   ],
   "dependencies": {
-    "bem-core": "2.5.0"
+    "bem-core": "^2.5.1"
```

Comment:
@dfilatov 
There was idea of meta-library which should provide one "official" tested version scope but that's not very cool to have strict versions for base libs as it may result in impossibility for `bower` to resolve deps when there are a few top level libs using different strict versions which are actually compatible. In Islands `^` is used.

Are you sure about this decision?